lib/bap_llvm/llvm_macho_loader.hpp

```diff
@@ -376,7 +461,13 @@ void symbols(const macho &obj, ogre_doc &s) {
     symbols(obj, sizes, s);
 }
 
+// It's safe to call getSymtabLoadCommand without any checks,
```

Comment:
is it true for all versions? I remember that somewhere in LLVM 3.4 we were getting a segmentation fault when we were asking for a symtab. It might be in ELF backend though.

Reply:
it's true for llvm versions >= 3.8

Reply:
OK, for the record, in case of 3.4 this function will not be used